packages/paste-core/components/button/__tests__/__snapshots__/button.test.tsx.snap

```diff
@@ -110,25 +107,24 @@ exports[`Button States Has a disabled state 1`] = `
   padding: 0.5rem 1rem;
   border-radius: 4px;
   font-size: 0.875rem;
-  line-height: 1.25rem;
+  line-height: 1.5rem;
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
   border: none;
   display: inline-block;
   outline: none;
   background: none;
-  -webkit-transition: background-color 100ms ease-in,border-color 100ms ease-in;
-  transition: background-color 100ms ease-in,border-color 100ms ease-in;
+  -webkit-transition: background-color 100ms ease-in,box-shadow 100ms ease-in;
+  transition: background-color 100ms ease-in,box-shadow 100ms ease-in;
   font-family: 'Whitney SSm A','Whitney SSm B','Helvetica Neue',Helvetica,Arial,sans-serif;
   font-weight: 500;
   cursor: not-allowed;
-  border-width: 2px;
-  border-style: solid;
   background-color: #e22525;
   color: #ffffff;
-  border-color: #ecb6b6;
+  box-shadow: inset 0 0 0 0.125rem #e22525;
   background-color: #ecb6b6;
+  box-shadow: inset 0 0 0 0.125rem #ecb6b6;
```

Comment:
Looks like the code added two box-shadow rules with different hexes.  Is this a bug?

Reply:
Good catch. I think I found the issue.

Reply:
Never mind. Confused now. We're rendering two different background colors there as well. https://github.com/twilio-labs/paste/pull/336/files#diff-56164225824111833313cb729bde7ffeL128

Reply:
Per Slack, seems this gets resolved in prod builds.